lib/bitex_bot/models/api_wrappers/api_wrapper.rb

```diff
@@ -15,29 +19,41 @@ class ApiWrapper
     :timestamp, # Integer
     :order # Actual order object
   ) do
-    def method_missing(method_name, *args)
-      order.send(method_name, *args)
+    def method_missing(method_name, *args, &_block)
+      order.send(method_name, *args) || super
```

Comment:
We should forward the block as well: `order.send(method_name, *args, &_block)`

Reply:
done